workbench-core/datasets/src/wbcDataSetsAuthorizationPlugin.test.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AddRemoveAccessPermissionRequest } from './models/addRemoveAccessPermissionRequest';
+import { GetAccessPermissionRequest } from './models/getAccessPermissionRequest';
+import { WbcDataSetsAuthorizationPlugin } from './wbcDataSetsAuthorizationPlugin';
+
+describe('wbcDataSetsAuthorizationPlugin tests', () => {
+  let plugin: WbcDataSetsAuthorizationPlugin;
+
+  const dataSetId: string = 'fake-dataset-id';
+  const subject: string = 'fake-group-id';
+
+  const accessPermission: AddRemoveAccessPermissionRequest = {
+    dataSetId: dataSetId,
+    permission: {
+      subject: subject,
+      accessLevel: 'read-only'
+    }
+  };
+
+  const getAccessPermission: GetAccessPermissionRequest = {
+    dataSetId: dataSetId,
+    subject: subject
+  };
+
+  beforeAll(() => {
+    const mockAuthService = {
+      init: jest.fn(),
+      isAuthorizedOnRoute: jest.fn(),
+      isAuthorizedOnSubject: jest.fn(),
+      isRouteIgnored: jest.fn(),
+      isRouteProtected: jest.fn(),
+      createGroup: jest.fn(),
+      deleteGroup: jest.fn(),
+      createIdentityPermissions: jest.fn(),
+      deleteIdentityPermissions: jest.fn(),
+      getIdentityPermissionsBySubject: jest.fn(),
+      removeUserFromGroup: jest.fn(),
+      addUserToGroup: jest.fn(),
+      getUserGroups: jest.fn(),
+      getGroupUsers: jest.fn(),
+      isUserAssignedToGroup: jest.fn(),
+      doesGroupExist: jest.fn(),
+      _groupManagementPlugin: jest.fn()
+    };
+
+    //@ts-ignore
+    plugin = new WbcDataSetsAuthorizationPlugin(mockAuthService);
+  });
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  describe('addAccessPermission tests', () => {
+    it('throws a notimplemented exception', async () => {
```

Comment:
I'm assuming these are just to make sure the code coverage trigger doesn't go off?